src/Microsoft.AspNetCore.Hosting/Startup/StartupExceptionPage.cs

```diff
@@ -267,10 +267,24 @@ private static void WriteException(Exception ex, StringBuilder builder, ref bool
                 }
             }
 
+            var exceptionMessage = HtmlEncodeAndReplaceLineBreaks(ex.Message);
+            var reflectionTypeLoadException = ex as ReflectionTypeLoadException;
+            if (reflectionTypeLoadException != null && reflectionTypeLoadException.LoaderExceptions.Length > 0)
+            {
+                var exceptionMessageBuilder = new StringBuilder(exceptionMessage);
+                exceptionMessageBuilder.Append("<br />LoaderExceptions:<ul>");
```

Comment:
Maybe make the Loader Exceptions a separate header: `<h3>Loader Exceptions</h3>`? It might help making it stand out from the exception text.